src/codeu/chat/server/View.java

```diff
@@ -78,6 +78,81 @@ public View(Model model) {
   @Override
   public Message findMessage(Uuid id) { return model.messageById().first(id); }
 
+  @Override
+  public Collection<String> userStatusUpdate(String name, Uuid owner) {
+
+    Collection<ConversationPayload> allConversations = all(model.conversationPayloadById());
+    Collection<String> contributions = new ArrayList<String>();
+
+    final User foundOwner = model.userById().first(owner);
+    final User foundUser = model.userByText().first(name);
+
+    if(foundUser != null) {
+      if(foundOwner.UserSet.contains(foundUser.id)) {
```

Comment:
foundOwner is null when a user with the specified name cannot be found in Model. The check in there is to prevent the following check from having errors (because it would check the current user's UserSet for a UUID from a User object that doesn't exist).